### DISCUSSION TITLE
Possible bug: `pocketbase migrate collections` doesn't properly save JSON field max size

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#4093] [Q&A]
I have a field named `vars` with a default max length of `20000`. However, it's not present in the collection dump. When I restore, the max size is `0`:

```json
 {
          "system": false,
          "id": "4fxgwtui",
          "name": "vars",
          "type": "json",
          "required": true,
          "presentable": false,
          "unique": false,
          "options": {}
        },
```

**Comment by ganigeorgiev** [comment]
It works fine for me locally. Make sure that you use PocketBase v0.20+.

**Reply by ganigeorgiev** [reply]
_If you are upgrading from an older version, also make sure that all v0.20+ migrations are applied (v0.20.0 comes with a system migration that will update older `json` fields with a default size - [1702134272_set_default_json_max_size.go](https://github.com/pocketbase/pocketbase/blob/master/migrations/1702134272_set_default_json_max_size.go))._

_edit: I noticed that the above migration [had one minor mistake](https://github.com/pocketbase/pocketbase/commit/28d15e86eb152038202950ddac2666ef05019436) but as noted in the commit even as it was it doesn't really matter because the old `json` fields didn't have any previous options anyway; it is creating one extra options struct unnecessary and it shouldn't have any effect to your problem._
